src/stories/Slider.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import { Story } from '@storybook/react';
+import React from 'react';
+import { Box } from '@mui/material';
+import Slider, { SliderProps } from '../components/Slider';
+
+export default {
+  title: 'Slider',
+  component: Slider,
+};
+
+const Template: Story<SliderProps> = (args) => {
+  return <Box height='100vh' display='flex' alignItems='center'>
+    <Slider {...args} />
+  </Box>;
+};
+
+export const Default = Template.bind({});
+
+export const SnapToMark = Template.bind({});
+
+const marks = [
+  {
+    label: '0%',
+    value: 0,
+  },
+  {
+    value: 25,
+  },
+  {
+    label: '50%',
+    value: 50,
+  },
+  {
+    value: 75,
+  },
+  {
+    label: '100%',
+    value: 100,
+  },
+];
+
+Default.args = {
+  defaultValue: 50,
+  min: 0,
+  max: 100,
+  step: 1,
```

Comment:
does this influence the snap? absence of step = snap to mark?

Reply:
Yes, it snaps to mark if `step` is missing

Reply:
Thanks 👍  makes sense - that's clever design by Mui. 
I noticed that it doesn't animate the drag.. guessing that's ok but may want to check with Chudo.